### PR TITLE
MAINT: updating project description

### LINKED ISSUE #125: Revise GitHub application title/label
Right now a Google search for "qiime 2" yields these top hits:

screen shot 2017-06-22 at 7 56 59 am

The description for q2studio could stand to be updated with a more accurate summary, or dropped altogether.

### PR DESCRIPTION
**Pull request by thermokarst**
Blocked by #129 

Fixes #125

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ***DISCLAIMER: This is a functional prototype of a graphical user interface for QIIME 2. This software is to be considered an alpha version, and may undergo drastic changes at any time.***
 
 # q2studio
-A web interface for QIIME 2.
+A graphical user interface for QIIME 2.
```
Comment (thermokarst) [inline README.md]: Need to also update the node description.

**Comment by gregcaporaso** [comment]
@thermokarst, there is a conflict that needs to be resolved here, and travis is failing.

**Comment by thermokarst** [comment]
Thanks @gregcaporaso, needed a rebase after #129 went in. Should be good to go now. Thanks!

**Comment by gregcaporaso** [comment]
Thanks @thermokarst!

**Pull request merged**